src/components/sections/Team.section.tsx

```diff
@@ -168,7 +168,7 @@ const Member: React.FC<MemberProps> = ({ name, title, profile }) => {
 const Department: React.FC<DepartmentProps> = ({ title, members }) => {
     return (
         <div>
-            <h3 className="mb-4 text-xl font-semibold capitalize md:mb-6 lg:mb-8 lg:text-2xl">
+            <h3 className="ext-2xl mb-6 text-2xl font-semibold capitalize md:mb-8">
```

Comment:
I believe this is a typo. "text-2xl" instead of "ext-2xl"

Reply:
Good catch